### PR TITLE
Ce3 migration

### PR DESCRIPTION
**Pull request by BrainHorse**
Closes https://github.com/evolution-gaming/smetrics/issues/171

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -6,13 +6,15 @@ object Dependencies {
   val prometheus       = "io.prometheus"        % "simpleclient"        % prometheusVersion
   val prometheusCommon = "io.prometheus"        % "simpleclient_common" % prometheusVersion
   val scalatest        = "org.scalatest"       %% "scalatest"           % "3.2.9"
-  val `cats-helper`    = "com.evolutiongaming" %% "cats-helper"         % "2.6.2"
-  val http4s           = "org.http4s"          %% "http4s-core"         % "0.22.7"
-  val doobie           = "org.tpolecat"        %% "doobie-core"         % "0.9.0"
+  // TODO replace with real version after migration https://github.com/evolution-gaming/cats-helper/pull/103
+  val `cats-helper`    = "com.evolutiongaming" %% "cats-helper"         % "2.6.3-SNAPSHOT"
```
Comment (DeviLab) [inline project/Dependencies.scala:10]: Use 2.7.3

Comment (t3hnar) [reply project/Dependencies.scala:10]: `3.0.0`

Comment (BrainHorse) [reply project/Dependencies.scala]: updated

**Comment by t3hnar** [comment]
released under `1.0.0`

**Pull request merged**